js/apps/admin-ui/src/clients/authorization/policy/Role.tsx

```diff
@@ -159,7 +145,6 @@ export const Role = () => {
             </Tbody>
           </Table>
         )}
-        {errors.roles && <FormErrorText message={t("requiredRoles")} />}
```

Comment:
This seems to be the only place the `requiredRoles` label was being used, so it can be removed from the properties files.